### PR TITLE
{vis}[dummy] ImageMagick (WIP)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2e66f0bbb307cfac3d2f for a full test report.

**Comment by fgeorgatos** [comment]
reviewed visually, that was easy ;-)

**Comment by boegel** [comment]
I'm not a big fan of merging this as is, since I *know* this is missing deps. I'll change this to `(WIP`)

File: easybuild/easyconfigs/i/ImageMagick/ImageMagick-6.9.0-9.eb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+version = '6.9.0-9'
+
+homepage = 'http://www.imagemagick.org/'
+description = """ImageMagick is a software suite to create, edit, compose, or convert bitmap images"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.imagemagick.org/download/']
+
```
Comment (boegel) [inline easybuild/easyconfigs/i/ImageMagick/ImageMagick-6.9.0-9.eb]: missing deps...

File: easybuild/easyconfigs/i/ImageMagick/ImageMagick-6.9.0-9.eb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '6.9.0-9'
+
+homepage = 'http://www.imagemagick.org/'
+description = """ImageMagick is a software suite to create, edit, compose, or convert bitmap images"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```
Comment (boegel) [inline easybuild/easyconfigs/i/ImageMagick/ImageMagick-6.9.0-9.eb]: should be a proper toolchain

**Comment by thiell** [comment]
I don't know for the missing deps, but this seems to work fine. I added the following module var as it is often defined for ImageMagick:

```
modextravars = { 'MAGICK_HOME': '%(installdir)s' }
```

**Comment by boegel** [comment]
@thiell: thanks for the suggestion, addded

The dependencies required for ImageMagick are probably available through the OS, but thanks for letting me know this works for you. Can you mention on which OS (or, use `eb --from-pr 1522 --upload-test-report`, cfr. https://github.com/hpcugent/easybuild/wiki/Review-process-for-contributions#automated-testing-of-easyconfigs-pull-requests)?

**Comment by thiell** [comment]
@boegel: for me, it worked on CentOS 6.7

**Comment by pescobar** [comment]
querying yum I got this list of dependencies
```
[escobar@login18 lcms2-2.7]$ yum deplist ImageMagick-devel |grep dependency|grep -v .so|grep -v config|grep -v bin/sh|grep -vi imagemagic|sort -u
  dependency: bzip2-devel
  dependency: freetype-devel
  dependency: ghostscript-devel
  dependency: jasper-devel
  dependency: lcms-devel
  dependency: libjpeg-devel
  dependency: libtiff-devel
  dependency: libX11-devel
  dependency: libXext-devel
  dependency: libXt-devel
```

**Comment by boegel** [comment]
better easyconfig (more complete list of deps) provided in #2438

**Comment by boegel** [comment]
made obsolete by #2438, thanks @pescobar!

**Pull request closed without merging**